addons/sourcemod/configs/szf/weapons.cfg

```diff
@@ -3,13 +3,14 @@
 	"general"
 	{
 		// "model"			REQUIRED - weapon's model that appears in the world. Two weapons can't share the same model.
-		// "rarity"			Weapon's rarity ("common", "uncommon", "rare", "pickup")
-		// "attrib"			Weapon's attributes, does nothing for pickups
-		// "text"			Text that appears in chat after picking up a weapon, does nothing for pickups
-		// "color"			Weapon's worldmodel color
-		// "callback"		On pickup callback (function must be public and be implemented in the plugin)
-		// "offset_origin"	Entity origin position offset, only works for pickups
-		// "offset_angles"	Entity angles offset, only works for pickups
+		// "rarity"			Weapon's rarity ("common", "uncommon", "rare", "pickup").
+		// "attrib"			Weapon's attributes, does nothing for pickups.
+		// "text"			Text that appears in chat after picking up a weapon, does nothing for pickups.
+		// "color"			Weapon's worldmodel color.
+		// "callback"		On pickup callback (function must be public and be implemented in the plugin).
+		// "offset_origin"	Entity origin position offset, only works for pickups.
+		// "offset_angles"	Entity angles offset, only works for pickups.
+		// "sound_pickup"	Pickup sound.
```

Comment:
really just "sound" should be enough and not restricted to pickups, don't see where else to use sound excluding voicelines, which that will be another time in seperate config with other sounds.